src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class `BalanceOperation` should not start with an empty line. Please remove the unnecessary empty line at the beginning of the class definition.